dev-util/edb-debugger/edb-debugger-1.0.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="edb is a cross platform x86/x86-64 debugger, inspired by Ollydbg"
+HOMEPAGE="https://github.com/eteran/edb-debugger"
+
```

Comment:
Please group it like this:

```
DESCRIPTION
HOMEPAGE
SRC_URI

LICENSE
SLOT
KEYWORDS
IUSE

DEPEND
RDEPEND
```

Reply:
If `RDEPEND` is second and inherits  from `${DEPEND}` (which includes virtual/pkgconfig), I get the following error:

```
RDEPEND.suspect               1
   dev-util/edb-debugger/edb-debugger-9999.ebuild: 'virtual/pkgconfig'
```

Until further info from you, I will keep the `xDEPEND`'s block intact.

Reply:
Sorry, I've pasted too far. RDEPEND/DEPEND use in any order you need.